source/web/HashingTools.js

```diff
@@ -0,0 +1,105 @@
+"use strict";
+
+function arrayBufferToHexString(arrayBuffer) {
+    var byteArray = new Uint8Array(arrayBuffer);
```

Comment:
Would be nice to have these as const and let now that we are updating?

Reply:
Yes! Agreed. Case of the copy+paste.. haha. I'll change it.